index.html

```diff
@@ -41,11 +41,63 @@
         background: white;
         padding: 0.5em;
       }
+      .translateme::before {
+        content: ' Translate this: ';
+        color: coral;
+        border-left: 4px solid coral;
+        padding-left: .5em;
+      }
+      .translateme {
+        background-color: #F3F3C3;
+      }
+      .checkme {
+        background-color: #BBEFFC;
+      }
+      .checkme::before {
+        content: ' Check this translation: ';
+        color: coral;
+        border-left: 4px solid coral;
+        padding-left: .5em;
+      }
     </style>
   </head>
 <body>
 <section id="abstract">
-  <p>A simple set of rules for placement of Ruby text in Japanese typography.</p>
+  <p class="checkme">This document describes simple method of processing for Ruby 
+  in Japanese layout realized with technologies like CSS, SVG, and XML-FO, 
+  as references to rendering engine implementors. 
+  Unlike 
+  <a href="https://www.w3.org/TR/jlreq/"><abbr title="Requirements for Japanese Text Layout">JLReq</abbr></a>
+  [[JLREQ]], only one layout method for each case is presented in this 
+  document, with consideration of best practices and important points in 
+  Japanese layout. 
+  Points took in consideration are described in 
+  <a href="#matters-considered-by-the-simple-placement-rules"></a>. 
+  Also layout of dual ruby composition is added in this document which is not 
+  described in [[JLREQ]].
+  </p>
+
+  <p class="checkme">In [[JLREQ]], which also targets to present various methods of layout 
+  used in the past, multiple or over-complexed ways of processing are 
+  described for one point. 
+  Expecially in processing of Ruby, number of cases need to be taken in 
+  consideration, and contradiction could happen for some combinations of 
+  requirements (see Note. "Protrusion of ruby from base characters").
```

Comment:
The "see..." can be turned into a link to the relevant section.

Reply:
thought that, but we need to modify note section to have unique ID... (let me work on this with ones in e.g. JLreq.

Reply:
filed as #22, to be done with adding unique IDs to note sections.